our-work/collaborations/musslick.yml

```diff
@@ -0,0 +1,15 @@
+title: Autonomous Empirical Research
+id: musslick
+description: The Autonomous Empirical Research Group (AER) strives to enhance and accelerate scientific discovery by automating each step of the empirical research process, from constructing a scientific hypothesis to conducting novel experiments. CCV's collaboration focuses on developing a framework which the group can use to run simulations to test machine learning models and run experiments with real participants.
+tags:
+  - active
+  - machine-scientist
```

Comment:
This tag seems a bit to specific... I would thing of a word that could include other groups at least in the future.Maybe just `machine-learning`? Also I think this collaboration should include `software-engineering`

Reply:
I've replaced machine-scientist with machine-learning. 
It might be worth thinking about whether we include both, as the term "machine scientist" is starting to be used in the literature (e.g. https://www.quantamagazine.org/machine-scientists-distill-the-laws-of-physics-from-raw-data-20220510). 
At the same time, "machine scientist" is perhaps just a synonym for "fancy curve fitting which produces interpretable models" or "one approach to interpretable ML."